src/main/kotlin/racingcar/controller/RaceController.kt

```diff
@@ -0,0 +1,49 @@
+package racingcar.controller
+
+import camp.nextstep.edu.missionutils.Randoms
+import racingcar.model.Car
+import racingcar.utils.Validator
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class RaceController {
+    private val cars = mutableListOf<Car>()
+
+    fun startRace() {
+        val carNames = InputView.getCarNames()
+        Validator.validateCarName(carNames)
+        createCars(carNames)
+
+        val moveCount = InputView.getMoveCount()
+        Validator.validateMoveCount(moveCount)
+        moveCars(moveCount)
+
+        OutputView.displayCarPositions(cars)
+        val winners = getWinners()
+        OutputView.displayWinners(winners)
+    }
+
+    private fun createCars(carNames: List<String>) {
+        cars.addAll(carNames.map { Car(it) })
+    }
+
+    private fun moveCars(moveCount: Int) {
+        for (i in 0 until moveCount) {
+            moveCarsForRound()
+        }
```

Comment:
```suggestion
        repeat(moveCount) {
            moveCarsForRound()
        }
```
```
for (_ in 0 until moveCount) {
    moveCarsForRound()
}
```
`repeat`이나 `_`를 사용해 불필요한 변수 `i`를 대체할 수 있습니다!

Reply:
@hongmyeoun 이 부분 다른 분께도 피드백을 받은 부분입니다. 저도 이번 과제를 하면서 for과 if를 남발했다고 생각했는데, repeat과 require이라는 좋은 함수들이 있더라구요! 피드백 감사합니다!